packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -30,57 +29,6 @@ type UserModelCreateParams = Omit<
  * @class {@link UserModel}
  */
 export default class extends EntityModel {
-  /**
-   * @todo: This method and `getUserByEntityId` is confusing. Should be fixed as part of:
-   *  https://app.asana.com/0/1200211978612931/1202937382769276/f
-   */
-  static async getUserByAccountId(
```

Comment:
This should have been removed in a prior PR, so have removed it here as a drive-by.